.devcontainer/docker-compose.yml

```diff
@@ -147,6 +124,25 @@ services:
     volumes:
       - ./meilisearch_data:/meili_data
 
+  umami:
+    container_name: umami
+    image: ghcr.io/umami-software/umami:postgresql-latest
```

Comment:
Should we set the postgres version? Since we've set 15 for the local db.

Reply:
This is the tag defined by umami, it does not relate to the version of postgres, basically they have a postgres and a mysql version for docker, and the tags are `postgres-latest` and `mysql-latest`